tests/Core/Components/Button/FluentButtonTests.razor

```diff
@@ -127,16 +127,19 @@
     }
 
     [Theory]
-    [InlineData(ButtonType.Button)]
-    [InlineData(ButtonType.Reset)]
-    [InlineData(ButtonType.Submit)]
-    public void FluentButton_TypeAttribute(ButtonType buttonType)
+    [InlineData(ButtonType.Button, "button")]
+    [InlineData(ButtonType.Reset, "reset")]
+    [InlineData(ButtonType.Submit, "submit")]
+    [InlineData(null, "")]
+    [InlineData((ButtonType)99, "")]
+    public void FluentButton_TypeAttribute(ButtonType? buttonType, string expectedValue)
     {
         // Arrange && Act
         var cut = Render(@<FluentButton Type="@buttonType">fluent-button</FluentButton>);
 
         // Assert
-        cut.Verify(suffix: buttonType.ToString());
```

Comment:
If you remove a `Verify` method, probably you need to delete the associated `verified.html` files 😊